python/grass/script/core.py

```diff
@@ -867,8 +867,8 @@ def _parse_opts(lines):
             break
         try:
             var, val = line.split(b"=", 1)
-        except ValueError:
-            raise SyntaxError("invalid output from g.parser: {}".format(line))
+        except ValueError as err:
+            raise SyntaxError("invalid output from g.parser: {}".format(line)) from err
```

Comment:
Now the code uses `as e`, `as err` and `as error`. I suggested to use `as error` in the new code (a single, existing word).

Reply:
I think there were some name collisions when only one name is used, but I'm not sure.

Reply:
Renaming these variables to be uniform would be something to do throughout the code.

Reply:
Hm, right, maybe not this PR.